src/pages/LP/Header/Header.jsx

```diff
@@ -8,31 +8,35 @@ import logo from '../../../images/lpLogo.png'
 
 
 
-export default function Header() {
-    const [queueCount, setQueueCount] = useState(0)
-
+export default function Header({socket}) {
     const { isAuthenticated } = useContext(AuthContext);
-    const { request } = useHttp();
-    const navigate = useNavigate();
+    const navigate = useNavigate(); 
+    const [queueCount, setQueueCount] = useState(null);
 
     useEffect(() => {
-        async function requestQueueCount() {
-            const data = await request(`${process.env.REACT_APP_API_URL}/api/landing/queuecount`, 'POST')
-            setQueueCount(data.count)
-        }
-        requestQueueCount();
-    },[])
-
-    function handleClick(e) { //delete andsunctionality
-        console.log('User authorized: ', isAuthenticated)
-        isAuthenticated ? navigate('/profile') : navigate('/signin')
-    }
+        socket?.emit('getQueueCount');
+        socket?.on('getQueueCount', (data) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
        socket?.on("getQueueCount", (data) => {
```